src/sentry/features/__init__.py

```diff
@@ -114,6 +114,7 @@
 default_manager.add("organizations:required-email-verification", OrganizationFeature, True)  # NOQA
 default_manager.add("organizations:rule-page", OrganizationFeature)
 default_manager.add("organizations:selection-filters-v2", OrganizationFeature, True)
+default_manager.add("organizations:session-replay", OrganizationFeature)
```

Comment:
If you add `True` as the 3rd argument here, you'll be able to control this via flagr in production.

Reply:
Oops, good to know. I'll follow up when we're closer to that.